spec/models/food_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
```

Comment:
- [ ] As required for the project, kindly make sure all your tests are passing as there are some of of hem which are not passing. I can also suggest you add tests to you controllers actions.
Here are the result of your tests:
![Screenshot from 2023-04-22 01-04-00](https://user-images.githubusercontent.com/83042742/233747001-490eb955-1f3d-476a-b228-956c6ccf7b72.png)